smart_open/smart_open_lib.py

```diff
@@ -36,6 +36,8 @@
 from ssl import SSLError
 import sys
 
+import errno
+from ntpath import split
```

Comment:
Why do you use ntpath instead of os.path?

Reply:
@mpenkov https://github.com/RaRe-Technologies/smart_open/pull/126#discussion_r132993646